Tests fail due to new feature in pylint

Tests are due to a new released check in pylint failing on this:
```
tests/test_portable.py:23:4: C0415: Import outside toplevel (webviz_app) (import-outside-toplevel)
```

--- a/tests/test_portable.py
+++ b/tests/test_portable.py
@@ -20,7 +20,7 @@ def test_portable(dash_duo, tmp_path):
 
     # Import generated app
     sys.path.append(str(appdir))
-    from webviz_app import app  # pylint: disable=import-error
+    from webviz_app import app  # pylint: disable=import-error, import-outside-toplevel
 
     # Start and test app
     dash_duo.start_server(app)

